### PR TITLE
Member form generator max

### PR DESCRIPTION
**Pull request by GingerBoy14**
I also have changes from Lena's branch - member

### CODE REVIEW

File: src/app/domains/Member/components/select/MemberSingleSelect/MemberSingleSelect.template.js [hunk 1]
```diff
@@ -1,8 +1,44 @@
...
-  return <div>MemberSingleSelect</div>
+  const { onChange } = props
+  const [currentUser, setCurrentUser] = useState('')
+  const [members, setMembers] = useState([])
+
+  useEffect(() => {
+    let tmpdata = []
+    db.collection('users')
+      .get()
+      .then((snapshot) => snapshot.forEach((doc) => tmpdata.push(doc.data())))
```
Comment (ARTfun00) [inline src/app/domains/Member/components/select/MemberSingleSelect/MemberSingleSelect.template.js:15]: 1. It's better to place tmpdata variable in the scope of promise callback (inside .then)
2. It's better to have more semantic name for this variable
3. Please check Array.map() method and refactor code here =)

File: src/app/domains/Member/components/views/MemberSimpleView/MemberSimpleView.template.js [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+      </Row>
+    </Container>
+  )
 }
 
-MemberSimpleView.propTypes = {}
-MemberSimpleView.defaultProps = {}
+MemberSimpleView.propTypes = {
+  withName: PropTypes.bool,
+  avatarUrl: PropTypes.string,
```
Comment (ARTfun00) [inline src/app/domains/Member/components/views/MemberSimpleView/MemberSimpleView.template.js:25]: Check code line #11 =)

File: src/app/services/getMembersForSelectComponent.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { db } from 'app/constants'
+
+const getMembersForSelectComponent = async () => {
+  const snapshot = await db.collection('users').get()
```
Comment (ARTfun00) [inline src/app/services/getMembersForSelectComponent.js:4]: same here... (check comment with collection names)

**Review by ARTfun00 — CHANGES_REQUESTED** [review]

File: src/app/domains/Color/components/views/ColorSimpleView/ColorSimpleView.template.js [hunk 4]
```diff
@@ -1,8 +1,29 @@
-const ColorSimpleView = (props) => {
-  return <div>ColorSimpleView</div>
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import { COLORS } from 'app/constants'
```
Comment (alexpassss) [inline src/app/domains/Color/components/views/ColorSimpleView/ColorSimpleView.template.js:3]: regroup imports

File: src/app/domains/Member/components/combined/MemberCombined/MemberCombined.template.js [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+import md5 from 'md5'
+
+const MemberCombined = (props) => {
+  const [open, setOpen] = React.useState(false)
+  const [openSnackbarSuccess, setOpenSnackbarSuccess] = React.useState(false)
+  const [openSnackbarError, setOpenSnackbarError] = React.useState(false)
+  const { register, handleSubmit, errors, control } = useForm()
+
+  const onSubmit = (data) => {
+    STORE.collection('users')
```
Comment (alexpassss) [inline src/app/domains/Member/components/combined/MemberCombined/MemberCombined.template.js:30]: create universal methods

File: src/app/domains/Member/components/combined/MemberCombined/MemberCombined.template.js [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+      </Snackbar>
+
+      <Dialog
+        fullWidth
+        maxWidth="xs"
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="form-dialog-title">
+        <Box p={2}>
+          <form onSubmit={handleSubmit(onSubmit)}>
```
Comment (alexpassss) [inline src/app/domains/Member/components/combined/MemberCombined/MemberCombined.template.js:86]: move form to the separate component

**Review by alexpassss — CHANGES_REQUESTED** [review]

File: src/app/domains/Member/components/combined/MemberCombined/MemberCombined.template.js (hunk 5 above)
Comment (GingerBoy14) [reply src/app/domains/Member/components/combined/MemberCombined/MemberCombined.template.js:30]: That will be fix when we create separate services

**Pull request closed without merging**